tests/lib/nested.sh

```diff
@@ -103,40 +109,44 @@ start_nested_core_vm_unit(){
     PARAM_SERIAL="-serial file:${WORK_DIR}/serial.log"
     PARAM_TPM=""
 
-    ATTR_KVM=""
-    if [ "$ENABLE_KVM" = "true" ]; then
-        ATTR_KVM=",accel=kvm"
-        # CPU can be defined just when kvm is enabled
-        PARAM_CPU="-cpu host"
-    fi
-
     # TODO: enable ms key booting for i.e. nightly edge jobs ?
-    OVMF_CODE=""
-    OVMF_VARS=""
+    VMF_CODE=""
+    VMF_VARS=""
     if [ "${ENABLE_SECURE_BOOT:-false}" = "true" ]; then
-        OVMF_CODE=".secboot"
+        VMF_CODE=".ms"
     fi
     if [ "${ENABLE_OVMF_SNAKEOIL:-false}" = "true" ]; then
-        OVMF_VARS=".snakeoil"
+        VMF_VARS=".snakeoil"
     fi
 
     mkdir -p "${WORK_DIR}/image/"
-    cp -f "/usr/share/OVMF/OVMF_VARS${OVMF_VARS}.fd" "${WORK_DIR}/image/OVMF_VARS${OVMF_VARS}.fd"
-    PARAM_BIOS="-drive file=/usr/share/OVMF/OVMF_CODE${OVMF_CODE}.fd,if=pflash,format=raw,unit=0,readonly=on -drive file=${WORK_DIR}/image/OVMF_VARS${OVMF_VARS}.fd,if=pflash,format=raw"
-    PARAM_MACHINE="-machine q35${ATTR_KVM} -global ICH9-LPC.disable_s3=1"
+    if os.query is-pc-amd64; then
+        QEMU_BIN=qemu-system-x86_64
+        PARAM_MACHINE="-machine q35${ATTR_KVM} -global ICH9-LPC.disable_s3=1"
+        PARAM_BIOS="-drive file=/usr/share/OVMF/OVMF_CODE${VMF_CODE}.fd,if=pflash,format=raw,unit=0,readonly=on -drive file=${WORK_DIR}/image/OVMF_VARS${VMF_VARS}.fd,if=pflash,format=raw"
+        TPM_DEVICE=tpm-tis
+        cp -f "/usr/share/OVMF/OVMF_VARS${VMF_VARS}.fd" "${WORK_DIR}/image/OVMF_VARS${VMF_VARS}.fd"
+    else
+        # Assume arm64
+        QEMU_BIN=qemu-system-aarch64
+        PARAM_MACHINE="-machine virt -cpu cortex-a57"
```

Comment:
PARAM_CPU? I see it is used other places.

Reply:
Good catch, changed now